documentation/modules/rhv-prerequisites.adoc

```diff
@@ -8,7 +8,7 @@
 
 The following prerequisites apply to {rhv-full} migrations:
 
-* You must have {rhv-short} admin permissions to be able to migrate any virtual machines. Alternatively, you need to have at least the `DiskCreator` and `UserVmManager` permissions granted by an `admin` user per each individual virtual machine you want to migrate.
+* You must have the {rhv-short} admin permissions to be able to migrate any virtual machines. Alternatively, you need to have at least the `DiskCreator` and `UserVmManager` permissions granted by an `admin` user per each individual virtual machine you want to migrate.
```

Comment:
looking better now, thanks.
few notes, I would emphasize that the permissions that needs to be granted are for the specific user that will perform the migration, something like:
`Alternatively, you need to have at least the `DiskCreator` and `UserVmManager` permissions set for the desired user per each individual virtual machine you want to migrate, this can be done by the admin user.`

Also I would drop the `you must have` part, and just display the two existing options, something like:
In order to migrate a virtual machine you may use admin account, which will give you access to  any virtual machine in the system. Alternatively ...`

Reply:
@bkhizgiy 
Please take a look now.